sdk/Sdk.Generators/Constants.cs

```diff
@@ -41,7 +41,8 @@ internal static class Types
             internal const string BindingPropertyNameAttribute = "Microsoft.Azure.Functions.Worker.Extensions.Abstractions.BindingPropertyNameAttribute";
             internal const string DefaultValue = "Microsoft.Azure.Functions.Worker.Extensions.Abstractions.DefaultValueAttribute";
 
-            internal const string HttpResponse = "Microsoft.Azure.Functions.Worker.Http.HttpResponseData";
+            internal const string HttpResponseData = "Microsoft.Azure.Functions.Worker.Http.HttpResponseData";
+            internal const string HttpResponseAttribute = "Microsoft.Azure.Functions.Worker.HttpResponseOutputAttribute";
```

Comment:
I definitely agree with moving away from `HttpResponse` as the name here. That felt like a potential hazard.

Just noting that we seem to be inconsistent about the conventions for these string member names and how they map back to the types. I would probably err on the side of more verbose names which are distinct from the types they represent. but if we want them all to match, we should make the second one here `HttpResponstOutputAttribute`.